### ISSUE TITLE
How can I disable the 2FA? It is asking me for 2FA code before the AAD consent screen. This is the extra step as per the Proctor.

### ISSUE THREAD

**Issue by Kedar782** [microsoft/fhir-proxy#90]
How can I disable the 2FA so I don't have to enter the code after the password? This is for the g(10) certification. I'm using the v2 branch, AAD and MSI.

I followed these steps but it did not work.

Go to the Azure Portal https://portal.azure.com/#home and sign in with the admin account for your organisation. Click on the hamburger icon to expand the menu and select Azure Active Directory. Select Properties from the left navigation menu. Click Manage security defaults, and toggle NO for Enable security defaults.

**Comment by vohuythao** [comment]
Hello @Kedar782 ,

Please login to Azure portal with Global Administrator account

Go to Azure Active Directory, then Users
![image](https://user-images.githubusercontent.com/14121587/205254475-a4b266cb-5955-4f69-b5fb-dc40c3721908.png)
You will see Per user MFA
![image](https://user-images.githubusercontent.com/14121587/205254632-4769de5e-d3c3-4c1b-9dd8-b6adb0129d3a.png)
It will open another tab, in this tab, you will see there is button to enable/disable each user MFA

Thank you

**Comment by Kedar782** [comment]
@vohuythao 

![image](https://user-images.githubusercontent.com/87669844/205341878-f96f6aff-6341-4774-80a6-e9d9c8299c3e.png)
![image](https://user-images.githubusercontent.com/87669844/205342071-999f28a1-099d-4fb8-97cc-5de50c53944b.png)

I changed the setting for 2FA. I have only one user for AD.
I have disabled 2FA for this account. But still, AD is asking me for 2FA every time.
Any idea what I'm doing wrong here?
